app/src/main/res/values/styles.xml

```diff
@@ -667,10 +667,14 @@
         <item name="behavior_halfExpandedRatio">0.001</item>
     </style>
 
-    <style name="TopSite.FaviconCard" parent="Mozac.Widgets.Favicon">
-        <item name="android:layout_width">@dimen/top_sites_card_size</item>
-        <item name="android:layout_height">@dimen/top_sites_card_size</item>
-        <item name="android:padding">@dimen/top_sites_card_padding</item>
+    <style name="TopSite.Card" parent="Mozac.Widgets.Favicon">
+        <item name="android:layout_width">@dimen/top_sites_card_width</item>
+        <item name="android:layout_height">@dimen/top_sites_card_height</item>
+        <item name="android:layout_marginTop">@dimen/top_sites_item_margin_top</item>
+        <item name="android:layout_marginBottom">@dimen/top_sites_item_margin_bottom</item>
```

Comment:
Can you point me to where these values are coming from? Usually we would want a top sites design to present a second row to know what these values should be

Reply:
margin top was introduced as a piece of feedback from Kate, since the top shadow was cut off. Adding a margin of to the top of the card to pad-out the item in grid was the way to fully show the shadow. 4 was the agreed upon value as per [this comment above](https://github.com/mozilla-mobile/fenix/pull/26614#issuecomment-1233189900) .

Margin bottom is the original value from Top Sites's original implementation in 2020, and seems to still be correct [per the figma](https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5536%3A189429).

Reply:
👍